pkg/cmd/server/kubernetes/master/master_config_test.go

```diff
@@ -25,6 +26,31 @@ import (
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 )
 
+var expectedGroupPreferredVersions []string = []string{
+	"apps/v1beta1,authentication.k8s.io/v1",
```

Comment:
if the order isn't significant (I can't remember if it is), then sort both lists